ada/ast.py

```diff
@@ -7755,6 +7765,13 @@ class ParamSpec(BaseFormalParamDecl):
 
     type_expression = Property(Entity.type_expr)
 
+    @langkit_property(public=True, return_type=Bool)
+    def is_constant_object():
+        """
```

Comment:
You only need this docstring on the base property (and same for the public annotation). Technically that's also true for the return type but you can leave it - especially since it will be mandatory in lkt